cmd/queue/main.go

```diff
@@ -250,7 +251,12 @@ func knativeProbeHandler(healthState *health.State, prober func() bool, isAggres
 			return
 		}
 
-		if preferPodForScaledown(logger, env.DownwardAPILabelsPath) {
+		if preferScaledown, err := preferPodForScaledown(env.DownwardAPILabelsPath); err != nil {
+			http.Error(w, "process downward API labels failed", http.StatusInternalServerError)
```

Comment:
This is a change in behavior! I'd actually say we should keep the logger in this function and just log the error as a warning if there was one. We should not prefer the pod for scaledown in this case, which this is effectively doing I think.

Reply:
In general I think we should just log the error and carry on. This is _optional_ functionality and we should not fail here.